### PR TITLE
Json response

### PR DESCRIPTION
**Pull request by saraialvarezrosas**

### CODE REVIEW

File: searchobject/src/main/java/org/jalau/at18/searchobject/controller/response/TokenResponse.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.jalau.at18.searchobject.controller.response;
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/controller/response/TokenResponse.java:1]: Please add documentation.

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java [hunk 2]
```diff
@@ -72,22 +75,27 @@ public void doFilter(ServletRequest request,
                 myReader.close();
 
                 //Verify that an empty or null file isn't entered
-                //The next line (75) Use for running the project.
-                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 100 && req.getPart("file").getContentType() != null) {
-                //if (res.getStatus() == 200) { //Use for running unit test of middleware package
+                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 161 && req.getPart("file").getContentType() != null) {
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: make the number 161 constant, like the magic numbers in Jose's project.

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java [hunk 3]
```diff
@@ -70,22 +75,19 @@ public void doFilter(ServletRequest request,
                 myReader.close();
 
                 //Verify that an empty or null file isn't entered
-                //The next line (73) Use for running the project.
-                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 100 && req.getPart("file").getContentType() != null) {
-                //if (res.getStatus() == 200) { //Use for running unit test of middleware package
-
+                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 161 && req.getPart("file").getContentType() != null) {
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: Change number 161 to a constant.

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java [hunk 4]
```diff
@@ -69,22 +73,19 @@ public void doFilter(ServletRequest request,
                 myReader.close();
 
                 //Verify that an empty or null file isn't entered
-                //The next line (73) Use for running the project.
-                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 100 && req.getPart("file").getContentType() != null  &&  req.getPart("file").getContentType().contains("zip")) {
-                //if (res.getStatus() == 200) { //Use for running unit test of middleware package
-
+                if (req.getPart("file").getSize() != 0L && req.getPart("file").getSize() > 161 && req.getPart("file").getContentType() != null  &&  req.getPart("file").getContentType().contains("zip")) {
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: Change the number 161 to constant.

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/token/GenerateToken.java [hunk 5]
```diff
@@ -8,6 +8,8 @@
  * Licence agreement you entered into with Jalasoft
  */
 import org.jalau.at18.searchobject.common.logger.At18Logger;
+import org.jalau.at18.searchobject.controller.response.ResponseMessage;
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/token/GenerateToken.java]: I believe this import is not necessary.

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java [hunk 6]
```diff
@@ -97,18 +105,23 @@ else if (tokenCounter < 1) {
...
+                throw new MiddlewareException(" Generated token ");
             }
         } catch (InstantiationError | MiddlewareException e) {
             LOG.warning(" ERROR LOADING THE MODEL " + e);
             e.printStackTrace();
-            PrintWriter out = response.getWriter();
             res.setStatus(400);
-            out.println ("    Status :    " + res.getStatus());
-            out.println (e.getMessage());
+            String x = Integer.toString( res.getStatus());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: Please rename this variable `x` to a name more descriptive

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java [hunk 7]
```diff
@@ -97,18 +105,23 @@ else if (tokenCounter < 1) {
...
+            ErrorResponse employee = new ErrorResponse(x, e.getMessage());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: Please rename this variable `employee` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java [hunk 8]
```diff
@@ -97,18 +105,23 @@ else if (tokenCounter < 1) {
...
+            String employeeJsonString = this.gson.toJson(employee);
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: Please rename this variable `employeeJsonString` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java [hunk 9]
```diff
@@ -94,18 +96,23 @@ public void doFilter(ServletRequest request,
...
+                throw new MiddlewareException("Generated token ");
             }
         } catch (InstantiationError | MiddlewareException e) {
             LOG.warning(" ERROR LOADING THE MODEL " + e);
             e.printStackTrace();
-            PrintWriter out = response.getWriter();
             res.setStatus(400);
-            out.println ("    Status :    " + res.getStatus());
-            out.println (e.getMessage());
+            String x = Integer.toString( res.getStatus());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: Please rename this variable `x` to a name more descriptive

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java [hunk 10]
```diff
@@ -94,18 +96,23 @@ public void doFilter(ServletRequest request,
...
+            ErrorResponse employee = new ErrorResponse(x, e.getMessage());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: Please rename this variable `employee` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java [hunk 11]
```diff
@@ -94,18 +96,23 @@ public void doFilter(ServletRequest request,
...
+            String employeeJsonString = this.gson.toJson(employee);
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: Please rename this variable `employeeJsonString` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java [hunk 12]
```diff
@@ -93,18 +94,23 @@ public void doFilter(ServletRequest request,
...
+                throw new MiddlewareException(" Generated token ");
             }
         } catch (InstantiationError | MiddlewareException e) {
             LOG.warning(" ERROR LOADING THE MODEL " + e);
             e.printStackTrace();
-            PrintWriter out = response.getWriter();
             res.setStatus(400);
-            out.println ("    Status :    " + res.getStatus());
-            out.println (e.getMessage());
+            String x = Integer.toString( res.getStatus());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: Please rename this variable `x` to a name more descriptive

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java [hunk 13]
```diff
@@ -93,18 +94,23 @@ public void doFilter(ServletRequest request,
...
+            ErrorResponse employee = new ErrorResponse(x, e.getMessage());
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: Please rename this variable `employee` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java [hunk 14]
```diff
@@ -93,18 +94,23 @@ public void doFilter(ServletRequest request,
...
+            String employeeJsonString = this.gson.toJson(employee);
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: Please rename this variable `employeeJsonString` to a name more descriptive according the project

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/token/GenerateToken.java (hunk 5 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/token/GenerateToken.java]: import was removed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java (hunk 14 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java (hunk 13 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java (hunk 12 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMidleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java (hunk 11 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java (hunk 10 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java (hunk 9 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java (hunk 8 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java (hunk 7 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: variable name changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java (hunk 6 above)
Comment (saraialvarezrosas) [reply searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java]: variable name changed

**Review by rodri-98 — APPROVED** [review]
Good job

**Review by DaniMB95 — APPROVED** [review]

**Pull request merged**